.github/workflows/tmp-fix-coderabbit-1750.yml

```
@@ -0,0 +1,118 @@
+          source = Path("src/codex/plugins-doctor.ts")
+          source_text = source.read_text(encoding="utf-8")
+          old_regex = r'''    const m = line.match(/^\s*([A-Za-z0-9_-]+)\s*=\s*("(?:\\.|[^"\\])*"|'[^']*'|[^#]+?)\s*(?:#.*)?$/);'''
+          new_regex = r'''    const m = line.match(/^\s*([A-Za-z0-9_-]+)\s*=\s*("(?:\\.|[^"\\])*"|'[^']*'|[^\s#]+)\s*(?:#.*)?$/);'''
+          if source_text.count(old_regex) != 1:
+              raise SystemExit("plugins-doctor matcher anchor did not match exactly once")
+          source.write_text(source_text.replace(old_regex, new_regex, 1), encoding="utf-8")
+
+          tests = Path("tests/codex-plugins-doctor.test.ts")
+          test_text = tests.read_text(encoding="utf-8")
+          old_block = r'''  test("malformed quoted marketplace values cannot wedge diagnosis", () => {
+    const { dir, configPath } = makeConfig(
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)